Makefile

```diff
@@ -93,7 +93,7 @@ generated-documentation:
 
 .PHONY: typedoc
 typedoc:
-	${ROOTDIR}/node_modules/.bin/typedoc index.ts --out ${ROOTDIR}/docs --excludeExternals --excludePrivate --excludeProtected --gitRevision main
+	${ROOTDIR}/node_modules/.bin/typedoc index.ts --out ${ROOTDIR}/docs --excludeExternals --excludePrivate --excludeProtected --gitRevision --gitRemote main
```

Comment:
You still want `--gitRevision main` but you want to add a `--gitRemote` value that points at the url of the repo.